README.md

```diff
@@ -4,22 +4,22 @@
 
 The **Progress Spark Toolkit** is a [CCS-compliant](https://github.com/progress/CCS) reference implementation from Progress meant to aid in creating an ABL application which provides RESTful API's for microservices. This repository primarily contains ABL artifacts and was built specifically for the **Progress Application Server for OpenEdge** to provide the back-end (server-side) support for exposing ABL logic via HTTP/HTTP.
 
-**Latest Release:** v6.0.0 (December 2020) for OE 11.7.7+ and OE 12.2.3+
+**Latest Release:** v6.0.1 (August 2021) for OE 11.7.10+ and OE 12.2.6+
 
 ## Requirements
 
-Due to the inclusion of the CCS libraries and new Business Logic features within the OpenEdge product, version **11.7.5** or **12.1** is required as a minimum, along with toolkit version **6.0.0** or later being highly recommended. Current builds of the PL files in the following distribution directories are compiled using the following OpenEdge versions for proper compatibility:
+Due to the inclusion of the CCS libraries and new Business Logic features within the OpenEdge product, version **11.7.8** or **12.2.4** is required as a minimum, along with toolkit version **6.0.1** or later being highly recommended. Current builds of the PL files in the following distribution directories are compiled using the following OpenEdge versions for proper compatibility:
```

Comment:
If (say) 12.2.6 is required, why are we including that source code (OE.BusinessLogic.*)? That code is already in the OE release.

Reply:
At risk of beating this horse yet again... There were some breaking changes in the 12.2.x code which makes some code no longer works well with 11.7. There is some modified code in there which uses a preprocessor check for version to make it work across versions. In terms of keeping the demos limping along this is unfortunately necessary.

Reply:
Attempted to be more specific on bare-minimums and WHY we need those. Still suggesting the latest LTS or supported release be used.